### ISSUE TITLE
The Android client is not accessible with screen readers anymore

### ISSUE THREAD

**Issue by emassey0135** [tailscale/tailscale#4165]
### What is the issue?

I have been trying to install the Tailscale Android client on a ChromeBook. I installed it from Google Play, but when I opened it ChromeVox did not read any controls when I tried to navigate with Search+Right arrow or using the touchscreen. I also tried using Commentary Screen Reader, and it did not read anything either. The version from F-Droid has the same behavior. I cannot try TalkBack because ChromeOS has a version of the Google Accessibility Suite with the ARC accessibility bridge instead of TalkBack.

### Steps to reproduce

I installed the latest version of Tailscale from Google Play on a ChromeBook, opened it with ChromeVox enabled, waited for it to load, and tried to navigate with ChromeVox commands and touchscreen gestures. I also did the same thing with Commentary Screen Reader enabled instead of ChromeVox.

### Are there any recent changes that introduced the issue?

According to the changelog, Talkback support was added in 1.20.0, and when I installed that version from APK Mirror, ChromeVox could read and interact with the controls, although there are still significant issues, for instance all the buttons are unlabeled. So something must have changed in between 1.20.0 and 1.22.0 that made the app inaccessible again.

### OS

Other

### OS version

Chrome OS dev channel, 101.0.4928.0

### Tailscale version

1.22.0 (Google Play)

**Comment by DentonGentry** [comment]
1.20 added Talkback but introduced a new crash which was widespread (https://github.com/tailscale/tailscale/issues/3754 plus a number of Play Store reviews that the app no longer worked at all). The 1.22 release went back to an earlier version to avoid the crash.

1.24 intends to update Gio again to an even newer version which fixes the crash, and contains Talkback support.

There is an Open Testing release track in the Play Store for the Tailscale app, which gets early release candidates. I built a 1.23.148 APK today which contains the Talkback fix that will be in 1.24. If you'd like to try it:

- from the Android device: https://play.google.com/store/apps/details?id=com.tailscale.ipn
- from the web (to enable your account to access the Open Testing track): https://play.google.com/apps/testing/com.tailscale.ipn
- 
If you want to go back to the regular production release track later, the Play Store will let you uninstall the Open Testing app. When you go back to install Tailscale again you'll get the choice of whether to use the Production or Open Testing version.

**Comment by DentonGentry** [comment]
Fixed in 1.24.0, released yesterday.
